src/controllers/shortener.controller.ts

```diff
@@ -0,0 +1,108 @@
+import { Request, Response } from 'express';
+import mongoose from 'mongoose';
+import { customAlphabet } from 'nanoid';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import ShortenedURL, { IShortenedURL } from '../models/shortener.model';
+
+// Create a custom nanoid generator for generating short URLs
+const nanoidGenerator = customAlphabet(
+  '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz',
+  5
+);
+
+interface CreateUrlBody {
+  originalUrl: string;
+  customName?: string;
+}
+
+export const createUrls = async (req: Request, res: Response): Promise<void> => {
+  const { originalUrl, customName } = req.body as CreateUrlBody;
```

Comment:
- The requirement expects you to receive url

```suggestion
  const { url, customName } = req.body as CreateUrlBody;
```